standalone/src/models/order/return-info/fields-config.ts

```diff
@@ -0,0 +1,33 @@
+import { fake, type TModelFieldsConfig } from '@/core';
+import { createRelatedDates } from '@/utils';
+import {
+  LineItemReturnItemRest,
+  LineItemReturnItemGraphql,
+} from '../line-item-return-item';
+import { TReturnInfoRest, TReturnInfoGraphql } from './types';
+
+const [returnDate] = createRelatedDates();
+
+const commonFieldsConfig = {
+  returnTrackingId: fake((f) => f.string.uuid()),
+  returnDate: fake(returnDate),
+};
+
+export const restFieldsConfig: TModelFieldsConfig<TReturnInfoRest> = {
+  fields: {
+    ...commonFieldsConfig,
+    items: fake((f) =>
+      f.helpers.arrayElement([LineItemReturnItemRest.random()])
+    ),
```

Comment:
`arrayElement` picks one random item from an array but here we just have one item so it doesn't seem its needed.

```suggestion
    items: fake((f) =>[LineItemReturnItemRest.random()]),
```